Controller/RegistrationController.php

```diff
@@ -96,7 +96,7 @@ public function registerAction(Request $request)
     /**
      * Tell the user to check his email provider.
      */
-    public function checkEmailAction()
+    public function checkEmailAction(Request $request)
```

Comment:
It's just that you wrote:

"However, if the controller is extended via a sub-class, and the methods of that class do need the request, **then there is no solution**."

Which is not correct :wink: